packages/wallets/src/features/cashier/modules/Transfer/hooks/useSortTransferAccounts.ts

```diff
@@ -0,0 +1,91 @@
+import { useMemo } from 'react';
+import { getMarketType } from '../helpers';
+import { TAccount, TAccountsList } from '../types';
+
+const useSortTransferAccounts = (accounts: TAccountsList) => {
+    return useMemo(
+        () => ({
+            tradingAccounts: accounts.tradingAccounts.sort(sortTradingAccounts),
+            walletAccounts: accounts.walletAccounts.sort(sortWalletsAccounts),
+        }),
+        [accounts]
+    );
+};
+
+export default useSortTransferAccounts;
+
+/** A custom hook that sort trading and wallet accounts to display on the screen. */
+const sortWalletsAccounts = (a: TAccount, b: TAccount) => {
+    if (!a?.accountName || !b?.accountName) return 0;
+    if (a.account_type === 'doughflow' && b.account_type === 'doughflow') {
+        return a.accountName.localeCompare(b.accountName);
+    } else if (a.account_type === 'crypto' && b.account_type === 'crypto') {
+        return a.accountName.localeCompare(b.accountName);
+    } else if (a.account_type === 'doughflow') {
+        // 'doughflow' comes first
+        return -1;
+    }
+    // 'crypto' comes next
+    return 1;
+};
+
+type TMt5LandingCompanyName = 'bvi' | 'labuan' | 'svg' | 'vanuatu';
```

Comment:
i'm pretty sure you can get the same typings `directly` from the API types - `THooks.MT5AccountsList['landing_company_short']`